service/http.proxy/src/test/scripts/org/kaazing/gateway/service/http/proxy/http.proxy.linked.proxy.reply.rpt

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Origin server
+#
+
+accept tcp://localhost:8030
+accepted
```

Comment:
@jitsni Hello, Jitu. I had initially opted for tcp since http k3po transport does not do an exact count match on headers (such as Via header). However, I have now done this change as per your suggestion as it does simplify the scripts.